Colibri.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Colibri : Ave, Muerte
+    {
+        //creamos variable para guardar informacion del constructor        
+           ModeloAnimal guardarinfo;
+
+        //se sobre escribe metodo comer
+        public override void comer()
+        {
+            Console.WriteLine("el colibri come");
+        }
+        //se crea comportmiento diferente
+        public void volar()
+        {
+            Console.WriteLine("el colibri vuela");
+        }
+
+        //se reciben los parametros de muerte y se calcula cuanto vivio
+        public string morir(int nacimiento, int fallecimiento, string causa)
+        {
+            int calculo;
+            calculo = nacimiento - fallecimiento;
```

Comment:
This is not the right approach to calculate age

Reply:
has been corrected